src/servicebus/azext_servicebus/_help.py

```diff
@@ -0,0 +1,476 @@
+"""
+
+helps['servicebus namespace authorizationrule create'] = """
+    type: command
+    short-summary: Creates Authorizationrule for the given Service Bus Namespace
+    examples:
+        - name: Creates Authorizationrule 'myauthorule' for the given Service Bus Namespace 'mynamepsace' in resourcegroup
+          text: az servicebus namespace authorizationrule create --resource-group myresourcegroup --namespace-name mynamespace
+           --name myauthorule --access-rights [Send, Listen]
+"""
+
+helps['servicebus namespace authorizationrule show'] = """
+    type: command
+    short-summary: Shows the details of Service Bus Namespace AuthorizationRule
+    examples:
+        - name: Shows the details of Service Bus Namespace AuthorizationRule
+          text: az servicebus namespace authorizationrule show --resource-group myresourcegroup --namespace-name mynamespace
+           --name myauthorule
+"""
+
+helps['servicebus namespace authorizationrule list'] = """
+    type: command
+    short-summary: Shows the list of AuthorizationRule by Service Bus Namespace
+    examples:
+        - name: Shows the list of AuthorizationRule by Service Bus Namespace
+          text: az servicebus namespace authorizationrule show --resource-group myresourcegroup --namespace-name mynamespace
+"""
+
+helps['servicebus namespace authorizationrule keys list'] = """
+    type: command
+    short-summary: Shows the connection strings of AuthorizationRule for the Service Bus Namespace
+    examples:
+        - name: Shows the connection strings of AuthorizationRule for the namespace.
+          text: az servicebus namespace authorizationrule list-keys --resource-group myresourcegroup --namespace-name mynamespace
+           --name myauthorule
+"""
+
+helps['servicebus namespace authorizationrule keys renew'] = """
+    type: command
+    short-summary: Regenerate the connection strings of AuthorizationRule for the Service Bus Namespace.
+    examples:
+        - name: Regenerate the connection strings of AuthorizationRule for the Service Bus Namespace.
+          text: az servicebus namespace authorizationrule regenerate-keys --resource-group myresourcegroup
+           --namespace-name mynamespace --name myauthorule --key PrimaryKey
+"""
+
+helps['servicebus namespace authorizationrule delete'] = """
+    type: command
+    short-summary: Deletes the AuthorizationRule of the Service Bus Namespace.
+    examples:
+        - name: Deletes the AuthorizationRule of the Service Bus Namespace.
+          text: az servicebus namespace authorizationrule delete --resource-group myresourcegroup --namespace-name mynamespace
+           --name myauthorule
+"""
+
+helps['sb queue create'] = """
+    type: command
+    short-summary: Creates the Service Bus Queue
+    examples:
+        - name: Creates Service Bus Queue.
+          text: az sb queue create --resource-group myresourcegroup --namespace-name mynamespace --name myqueue
+"""
```

Comment:
There's no command `az sb queue create`.
Should this be `az servicebus queue create`?

Reply:
Resolved